### ISSUE TITLE
Custom logo or text?

### ISSUE THREAD

**Issue by joshuapinter** [soffes/Clock.saver#13]
First off, let me start by saying that this screensaver is beautiful. It's perfectly executed and I'm seriously impressed. We have it running (mandatory) for all the computers in our office and it looks elegant AF.

![img_0091 4](https://user-images.githubusercontent.com/180819/50192597-2bb1f400-02f0-11e9-8c9a-69f65e9ae290.jpeg)

My questions arises around having an option to show a custom logo or piece of text on the screensaver. I don't want to ruin the aesthetic, but perhaps text or a logo in the same vein as the braun logo. 

Let me know your thoughts on this and if it's something you have considered. Since this is open source, it might be something we can build a PR for, but I wanted to get your input on it before we undertake that effort. 

Thanks!

**Comment by soffes** [comment]
I'm not super into the idea but if you want to PR it, I'd be happy to take a look.

**Comment by joshuapinter** [comment]
Ended up doing this just to see how it felt in practice. Hard coded our logo into the screensaver. I found anything more than this, like our work mark, made it feel too cluttered. 

<img width="1440" alt="screen shot 2018-12-31 at 11 13 30 pm" src="https://user-images.githubusercontent.com/180819/50570704-58cfbf00-0d52-11e9-9eaa-f2c04fab99ee.png">

Certainly not as nice as the classic BRAUN logo but it's not bad. 

Will see how it goes over the next bit and see if it's worth making this configurable. My feeling is if anybody really wanted to do it, they would just modify the code itself. No need to monkey around with configuration. 

You've built a clean and simple screensaver that honours Dieter Rams. I think it should be kept that way. 

👍
